src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,35 @@
+<div class="footer">
```

Comment:
```suggestion
<footer class="footer">
```
セマンティック要素を意識的に使えるとより良くなると思います 👍